insights/specs/default.py

```diff
@@ -950,6 +950,7 @@ def is_mod_loaded_for_ss(broker):
     subscription_manager_installed_product_ids = simple_command("/usr/bin/find /etc/pki/product-default/ /etc/pki/product/ -name '*pem' -exec rct cat-cert --no-content '{}' \;")
     swift_object_expirer_conf = first_file(["/var/lib/config-data/puppet-generated/swift/etc/swift/object-expirer.conf", "/etc/swift/object-expirer.conf"])
     swift_proxy_server_conf = first_file(["/var/lib/config-data/puppet-generated/swift/etc/swift/proxy-server.conf", "/etc/swift/proxy-server.conf"])
+    sysconfig_grub = simple_file("/etc/sysconfig/grub")
```

Comment:
better collect the original /etc/default/grub file, since the /etc/sysconfig/grub is a symlink to it, in case the symlink is broken.

Reply:
`first_file` can be used in this case to indicate that they are the same

```
sysconfig_grub = first_file(["/etc/default/grub", "/etc/sysconfig/grub"])
```